test/edwadrds_point_test.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:convert/convert.dart';
+import 'package:solana/src/edwards25519/compressed_edwards_y.dart';
+import 'package:solana/src/edwards25519/edwards_point.dart';
+import 'package:solana/src/edwards25519/field_element.dart';
+import 'package:test/test.dart';
+
+EdwardsPoint ed25519BasePoint = EdwardsPoint(
+  FieldElement([
+    -14297830,
+    -7645148,
+    16144683,
+    -16471763,
+    27570974,
+    -2696100,
+    -26142465,
+    8378389,
+    20764389,
+    8758491
+  ]),
+  FieldElement([
+    -26843541,
+    -6710886,
+    13421773,
+    -13421773,
+    26843546,
+    6710886,
+    -13421773,
+    13421773,
+    -26843546,
+    -6710886,
+  ]),
+  FieldElement([1, 0, 0, 0, 0, 0, 0, 0, 0, 0]),
+  FieldElement([
+    28827062,
+    -6116119,
+    -27349572,
+    244363,
+    8635006,
+    11264893,
+    19351346,
+    13413597,
+    16611511,
+    -6414980,
+  ]),
+);
+
+void main() {
+  /**
```

Comment:
Dart-style comments should be `///`.

Reply:
Ok, didin't know. It does understand `/**/` so not sure why they put that if they want `///`